src/defs.h

```diff
@@ -761,6 +762,7 @@ struct user_params_t {
     bool withsock_accl = false;
     int max_looping_over_recv = 1;
     int tos = 0x00;
+    int tcp_connect_timout_ms = 5000;
```

Comment:
Do you see any reason to change default parameter from `1000ms` to `5000ms`?

Reply:
I think it was 1 millisec before. There are slow setups that need more than 1 second to respond. Real setup case from 3278795. So i'd like to give some reasonable time to complete the connection establishment. 1 second just not always enough.